lift/parallel.h

```diff
@@ -350,6 +350,35 @@ struct parallel
                                            UniqueOutputIterator unique_keys_output,
                                            LengthOutputIterator run_lengths_output,
                                            allocation<system, uint8>& temp_storage);
+    //Fills an allocation with a value
+    /**
+     * Parallel for-each implementation. Sets each element in range [begin, end[ to value.
+     * \tparam InputIterator    Iterator type
+     * \tparam T                Type for input data.
+     *
+     * \param begin             Iterator pointing at the first element to be processed.
+     * \param end               Iterator pointing at the end of the range to be processed.
+     * \param value             The value with which to fill the range.
+     * \param launch_parameters Grid launch parameters for GPU backend.
+     */
+    template <typename InputIterator, typename T>
+    static inline void fill(InputIterator begin,
+                            InputIterator end,
+                            T value,
+                            int2 launch_parameters = { 0, 0 });
```

Comment:
It probably doesn't make sense to expose launch parameters for this. We should be able to figure out the optimal grid config since we know ahead of time what kernel we'll run.